src/main/java/com/dawsonsystems/session/MongoManager.java

```diff
@@ -552,13 +577,18 @@ private void initDbConnection(String path) throws LifecycleException {
               .retryReads(false)
               .retryWrites(false);
 
-      List<MongoCredential> mongoCredentials = new ArrayList<>();
-
-      if (sslTrustStorePem != null && sslTrustStorePem.length() > 0 && sslKeyStorePem != null && sslKeyStorePem.length() > 0) {
+      Optional<SSLContext> sslContext = null;
+      if (Boolean.parseBoolean(useJndiSslContext)) {
```

Comment:
I'd either add a null check for `useJndiSslContext` or default the field to "false" at the top of the class

Reply:
i think Boolean.parseBoolean will return false if null